lld/MachO/ObjC.cpp

```diff
@@ -349,11 +349,15 @@ void objc::checkCategories() {
 namespace {
 
 class ObjcCategoryMerger {
+  // In which language was a particular construct originally defined
+  enum SourceLanguage { SourceObjC, SourceSwift, SourceUnknown };
```

Comment:
```suggestion
  enum SourceLanguage { ObjC, Swift, Unknown };
```
What do you think about removing the redundant prefix?

Reply:
This was my original naming intent, but the names are too generic and might be confusing to what they are actually referring too when used later on.

Reply:
As long as these values are always prefixed with `SourceLanguage::`, I don't think it would be confusing.